doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex

```diff
@@ -183,6 +183,475 @@ \subsubsection{Outputs}\label{outputs-015}
+
+
+
+\subsubsection{Inputs}\label{inputs}
+
+\paragraph{Field: Name}\label{field-name-06}
+
+This alpha field contains the identifying name for the coil system cooling water object. Any reference to this coil system by another object will use this name.
+
+\paragraph{Field: Air Inlet Node Name}\label{field-air-inlet-node-name-01}
+
+This alpha field contains the coil system cooling water object air inlet node name.
+
+\paragraph{Field: Air Outlet Node Name}\label{field-air-outlet-node-name-01}
+
+This alpha field contains the coil system cooling water object air outlet node name.
+
+\paragraph{Field: Availability Schedule Name}
+
+This alpha field contains the schedule name which contains information on the availability of the coil system cooling water object to operate. A schedule value equal to 0 denotes that the coil system must be off for that time period. A value greater than 0 denotes that the coil system is available to operate during that time period. This schedule may be used to completely disable the coil system as required. If this field is left blank, the schedule has a value of 1.
+
+\paragraph{Field: Cooling Coil Object Type}\label{field-cooling-coil-object-type-01}
+
+This alpha field contains the identifying type of cooling coil specified in the coil system cooling water object. Valid choices for this field are:
+\begin{itemize}
+\item Coil:Cooling:Water
+\item Coil:Cooling:Water:DetailedGeometry
+\item CoilSystem:Cooling:Water:HeatExchangerAssisted
+\end{itemize}
+
+
+\paragraph{Field: Cooling Coil Name}\label{field-cooling-coil-name-01}
+
+This alpha field contains the identifying name given to the coil system water cooling coil.
+
+\paragraph{Field: Dehumidification Control Type}
+
+This alpha field contains the type of dehumidification control. The following options are valid for this field:
+
+\textbf{None} - meet sensible load only, no active dehumidification control. The default is None.
+
+\textbf{Multimode} - activate water coil and meet sensible load. If no sensible load exists, and Run on Latent Load = Yes, and a latent load exists, the coil will operate to meet the latent load. If the latent load cannot be met the heat exchanger will be activated. IF Run on Latent Load = No, the heat exchanger will always be active. This control mode either switches the coil mode or allows the heat exchanger to be turned on and off based on the zone dehumidification requirements. Valid only with cooling coil type CoilSystem:Cooling:Water:HeatExchangerAssisted.
```

Comment:
Similar edits to the idd notes:
~IF Run on Latent Load = No, the heat exchanger will always be active.~ (@rraustad This isn't really what happens, is it?)
based on the ~zone~ dehumidification ~reqiurements~ *setpoint*.

Reply:
The HX is always active if the Dehumidification control is **None** or **CoolReheat**. When it is **Multimode**, the  HX is turned on only if the dehumidification load cannot be met without the HX.

Reply:
Ok, so it really doesn't have anything to do with the value of Run on Latent, but rather what the humidity control type is. So, please make sure the notes and the I/O Ref reflect that.

Reply:
OK. Made changes to the idd and IO ref.  I expect @rraustad to review and revise it if he doesn't agree.